src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/ProjectSystem/VS/Input/Commands/EditCsprojCommand.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.VisualStudio.Editor;
+using Microsoft.VisualStudio.Packaging;
+using Microsoft.VisualStudio.ProjectSystem.Input;
+using Microsoft.VisualStudio.ProjectSystem.Utilities;
+using Microsoft.VisualStudio.ProjectSystem.VS.Utilities;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Text;
+using System;
+using System.ComponentModel.Composition;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
+{
+    [ProjectCommand(ManagedProjectSystemPackage.DotnetProjectSystemCommandSet, ManagedProjectSystemPackage.EditProjectFileCmdId)]
```

Comment:
>DotnetProject [](start = 48, length = 13)

We should call this ManagedProjectSystemCommandSet to be consisten in the usage of managed vs dotnet